src/autodiscovery.ts

```diff
@@ -543,7 +419,7 @@ export class AutoDiscovery {
         }
     }
 
-    private static fetch(resource: URL | string, options?: RequestInit): ReturnType<typeof global.fetch> {
+    public static fetch(resource: URL | string, options?: RequestInit): ReturnType<typeof global.fetch> {
```

Comment:
Given that AutoDiscovery is part of the js-sdk's public API, making this public really isn't great. Can you find any other way of doing it? (Is it just to support replacing `fetchFn` in the tests? I don't really follow why we can't do what we need with fetch-mock.)

At the very least it needs documenting, but exposing this to applications feels like something we should avoid.